index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>assignment01</title>
+    <script src="jquery-3.4.1.min.js"></script>
+<script src="script.js"></script>
+</head>
+<body>
+    <div id="test"></div>
+    <div class="mcc"></div>
+    <h1>Greetings</h1>
+    <p>Hello, my name is <span id="first-name">Donjeta</span><span id ="last-name">Emerllahu</span>!</p>
```

Comment:
The idea with this assignment is that the HTML won't actually contain your names or hobbies - but that the content will be added with your jQuery file. So this file should look pretty much identical to the one pictured in the assignment.